spec/models/setting_spec.rb

```diff
@@ -24,11 +24,11 @@
     describe setter do
       subject { -> { Setting.send(setter, blob) } }
       it "sets #{getter}" do
-        it_should change { Setting.send(getter) }.from(nil).to(blob)
+        will change { Setting.send(getter) }.from(nil).to(blob)
```

Comment:
I don’t really see how using `will` reads more natural in this case? What’s the reason against the default `expect` style that we use in other places?

```suggestion
        expect(subject.call).to change { Setting.send(getter) }.from(nil).to(blob)
```